scripts/scripts.js

```diff
@@ -683,6 +683,16 @@ async function loadLazy(doc) {
     decorateGuideTemplateCodeBlock();
   }
 
+  // labs banner
```

Comment:
Just wondering if this may introduce a bit of CLS as the labs block appears.
<img width="946" alt="image" src="https://github.com/adobe/helix-website/assets/1191451/db2fbd52-27fa-4456-ba36-0b198d467b0e">

Reply:
True... maybe we should overlay the hero image?

Reply:
@sdmcraft Moving the creation of the labs block to the egaer phase and letting the block loader do its thing removed the CLS.